api-version-compatibility-config.yaml

```diff
@@ -0,0 +1,24 @@
+# The configuration file is used by the GitHub Actions workflow defined in .github/workflows/check-api-compatibility.yaml.
+# It is used to define exclusions when comparing different versions of CRD schemas and
+# specifies the fields that should be excluded from the comparison for each version of a CRD.
+# The structure of the file is as follows:
+#
+# <crd-filename>:
+#   exclusions:
+#     <version>:
+#       - <exclusion-path>
+
+operator.kyma-project.io_kymas.yaml:
+  exclusions:
+    v1beta1:
+      - .spec.properties.sync
+    v1beta2:
+      - .spec.properties.modules.x-kubernetes-list-map-keys
+      - .spec.properties.modules.x-kubernetes-list-type
+operator.kyma-project.io_moduletemplates.yaml:
+  exclusions:
+    v1beta1:
+      - .spec.properties.target
+      - .spec.required
+    v1beta2:
+      - .spec.required
```

Comment:
```suggestion
# The configuration file is used by the GitHub Actions workflow defined in .github/workflows/check-api-compatibility.yaml.
# It is used to define exclusions when comparing different versions of CRD schemas and
# specifies the fields that should be excluded from the comparison for each version of a CRD.
# The structure of the file is as follows:
#
# <crd-filename>:
#   exclusions:
#     <version>:
#       - <exclusion-path> (may be a yq operator without spaces)

operator.kyma-project.io_kymas.yaml:
  exclusions:
    v1beta1:
      - .spec.properties.sync
    v1beta2:
      - .spec.properties.modules.x-kubernetes-list-map-keys
      - .spec.properties.modules.x-kubernetes-list-type
operator.kyma-project.io_moduletemplates.yaml:
  exclusions:
    v1beta1:
      - .spec.properties.target
      - .spec.required[]|select(.=="target")

```

We can actually also supply a yq operator to the `del` operator and explicitly remove the "target" entry from the required array. We only can't use spaces as the parsing of the exlucsions does not properly detect this.

Reply:
I tried do that but I didn't work, but your the no space trick of yours should work